src/main/java/hudson/plugins/promoted_builds/conditions/ManualCondition.java

```diff
@@ -52,8 +59,9 @@
  * @author Peter Hayes
  */
 public class ManualCondition extends PromotionCondition {
-    private String users;
+    private static String users;
```

Comment:
By making this `static`, you have forced all `ManualCondition` instances to use the same list of users, which is causing the issue noted in your email to jenkinsci-dev.

Reply:
thank you very much for the direction!
I'll give it a try.